src/pages/HomePage.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Filter from '../components/Filter';
+import Header from '../components/Header';
+import CountryItem from '../components/CountryItem';
+import { loadCountries } from '../redux/countries/countries';
+import '../styles/HomePage.css';
```

Comment:
- [ ] For all your pages, kindly ensure you maintain the font style as we have on the design template.

  Your font style | Design font style
  :---:|:---:
  ![dkkd](https://user-images.githubusercontent.com/36542986/210246806-e03155bb-cd31-4e49-a6aa-495d3268daa3.jpg) | ![idoee](https://user-images.githubusercontent.com/36542986/210246801-489b4435-b984-43ce-9c80-25028d455b31.jpg)